cmake/usFunctionCheckResourceLinking.cmake

```diff
@@ -16,7 +16,7 @@ function(usFunctionCheckResourceLinking)
       set(_suffix .rc)
     elseif(UNIX)
       execute_process(
-        COMMAND ${CMAKE_LINKER} -r -b binary -o "${CMAKE_CURRENT_BINARY_DIR}/us_resource_link.o" "${CMAKE_COMMAND}"
+        COMMAND ${CMAKE_LINKER} -r -o "${CMAKE_CURRENT_BINARY_DIR}/us_resource_link.o" ${_ADDITIONAL_LINKER_FLAGS} --format=binary "${CMAKE_COMMAND}"
```

Comment:
Didn't notice this defined in these files. Must be someplace else in the build files?

Reply:
_ADDITIONAL_LINKER_FLAGS is defined in the top-level CMakeLists.txt file